src/server.js

```diff
@@ -91,6 +97,7 @@ function index(req, res, next) {
 			config
 		);
 		var template = _.template(file);
+		res.setHeader("Content-Security-Policy", "default-src *; style-src * 'unsafe-inline'; script-src 'self'; reflected-xss block");
```

Comment:
> I'm not sure there's any need to send CSP on all requests (all other requests besides index.html would be static content).

Yeah OK, we'll adjust later if necessary.
I know that we essentially only send index and then the websocket takes over, but I'm thinking that other fallbacks from Socket.io might be worth "policying". But again, I don't think Socket.io would send this policy even if we set if with all requests (because it does go through Express).

> > reflected-xss : Instructs a user agent to activate or deactivate any heuristics used to filter or block reflected cross-site scripting attacks, equivalent to the effects of the non-standard X-XSS-Protection header.

OK so basically it's unnecessary in Chrome because filters are already enabled, but I don't know for other browsers.

> I've also didn't set extra hardened directives because I don't know what we would want to allow for eventual plugins to do.

Agreed, but what about hardening at the moment and adapting when packages are a reality (or start to be one)?
After all, it's not set in stone, and some directives of the policy might not change anyway (especially `connect-src` and `form-action`, maybe `child-src`).

> Explicitness. We could eventually set it to `self` instead, and configure other directives as needed.

Good point, to keep in mind.